src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -534,9 +535,9 @@ const WorkspaceDashboard = _.flow(
         h(InfoRow, { title: 'Cloud Name' }, [
           h(AzureLogo, { title: 'Microsoft Azure', role: 'img', style: { height: 16 } })
         ]),
-        // h(InfoRow, { title: 'Location' }, [
-        //   h(TooltipCell, !!storageLocation ? [getRegionFlag(storageLocation), ' ', getRegionLabel(storageLocation)] : ['Loading'])
-        // ]), depends on TOAZ-265
+        h(InfoRow, { title: 'Location' }, [
+          h(TooltipCell, !!storageLocation ? [getRegionFlag(storageLocation), ' ', getRegionLabel(storageLocation)] : ['Loading'])
```

Comment:
`getRegionFlag` and `getRegionLabel` currently only supports useast. @mlilynolting are you adding more regions to that structure?

![image](https://user-images.githubusercontent.com/484484/215523649-864549b8-d798-4a9e-874d-c6bf7ed0aaf0.png)

Reply:
Yes, I'm adding those regions. I have no issue with falling back on the raw region value for display as of this PR.